### PR TITLE
refactor(HMS-4895): use -oci-tag wherever possible

### PR DESCRIPTION
**Pull request by avisiedo**
Prepare the change for adding the unit test task, by migrating the tasks to -oci-ta which does not require PVC and the ephemeral information is trusted.

This change apply to the pullrequest tekton pipeline.

https://issues.redhat.com/browse/HMS-4556

---

TODO

- [ ] Trigger build pipeline for this PR to check it runs successfully before merging.

### CODE REVIEW

**Comment by frasertweedale** [comment]
Maybe rebase to see if the Konflux error goes away?

**Comment by frasertweedale** [comment]
The commit messages refers to `-oci-tag`, but I think it should be `oci-ta` right?

**Comment by frasertweedale** [comment]
@avisiedo please rebase on latest `main` and merge when green.

File: .tekton/idmsvc-backend-pull-request.yaml [hunk 1]
```diff
@@ -446,20 +472,24 @@ spec:
...
           value: task
         resolver: bundles
-      workspaces:
-      - name: workspace
-        workspace: workspace
+      # workspaces:
+      # - name: git-auth
+      #   optional: true
+      # - name: netrc
+      #   optional: true
```
Comment (frasertweedale) [inline .tekton/idmsvc-backend-pull-request.yaml:492]: is it better just to delete these lines, or do we need them commented for some reason?  If we want to keep them, please add a comment why it's important to see this :)

Comment (avisiedo) [reply .tekton/idmsvc-backend-pull-request.yaml]: thanks! I have removed them!

**Review by frasertweedale — APPROVED** [review]

**Pull request merged**